StaticAnalyzer/views/android/VirusTotal.py

```diff
@@ -21,7 +21,14 @@ def get_report(self, file_hash):
             }
             headers = {"Accept-Encoding": "gzip, deflate"}
             try:
-                response = requests.get(url, params=params, headers=headers)
+                if settings.UPSTREAM_PROXY_ENABLED:
+                    if settings.UPSTREAM_PROXY_USERNAME is None :
+                        proxies = {"https": settings.UPSTREAM_PROXY_IP + ":" + settings.UPSTREAM_PROXY_PORT}
```

Comment:
Better to add a settings to switch between `https` and `http` ?

Reply:
will add support for https and http